### DISCUSSION TITLE
笔记丢失

### DISCUSSION THREAD

**Discussion by Schemingboy** [Quorafind/Obsidian-Thino#725] [Q&A 问答]
更新到最新版（需要登录）后，如何找到我之前在 thino 记的笔记？

**Comment by cpy23** [comment]
我把仓库位置转移之后，再打开thino。之前的笔记也都不见了？ 要怎么找回啊

**Comment by juestchaos** [comment]
thino 目前看并不会丢失笔记
在 thino 的 FAQ 提到过，有用户转移仓库位置，路径变化，或者仓库的日记变化，以及thino 设置读取的文件位置变化
导致了看不到数据读取到 thino 列表
如果你迁移了数据，那么请进行检查自己的日记路径，或者说检查自己的存储文职路径，是否在 thino 和 ob 中设置正确

**Comment by cpy23** [comment]
明白了，谢谢回复。后面使用，按你的提示，注意路径和位置，应该就OK了

________________________________
发件人: JuestChaOS ***@***.***>
发送时间: 2024年12月9日 10:16
收件人: Quorafind/Obsidian-Thino ***@***.***>
抄送: cpy23 ***@***.***>; Comment ***@***.***>
主题: Re: [Quorafind/Obsidian-Thino] 笔记丢失 (Discussion #725)


thino 目前看并不会丢失笔记
在 thino 的 FAQ 提到过，有用户转移仓库位置，路径变化，或者仓库的日记变化，以及thino 设置读取的文件位置变化
导致了看不到数据读取到 thino 列表
如果你迁移了数据，那么请进行检查自己的日记路径，或者说检查自己的存储文职路径，是否在 thino 和 ob 中设置正确

―
